stories/amo/HeroRecommendation.js

```diff
@@ -46,6 +48,19 @@ storiesOf('HeroRecommendation', module)
             title: 'without image',
             sectionFn: () => render({ featuredImage: null }),
           },
+          {
+            title: 'with error',
+            sectionFn: () =>
+              render(
+                {},
+                {
+                  errorHandler: new ErrorHandler({
+                    id: 'some-id',
+                    capturedError: new Error('Some error'),
```

Comment:
This doesn't work for some reason, and ends up generating an error from the `ErrorList` component when it tries to render it. I'm not sure why because, AFAICT it's doing the exact same thing as `createStubErrorHandler`, which I had trouble using here because `sinon` doesn't exist.